### PR TITLE
tlt-2913: qualtrics_link - text updates for unauthorized message

### PR DESCRIPTION
**Pull request by elliottyates**

### CODE REVIEW

File: qualtrics_link/templates/qualtrics_link/_unauthorized_message.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h3>
+  We're sorry, your ID doesn't appear to have access to this Qualtrics license.
+</h3>
+<p>
+  <ul>
+    <li>
+      HMS/HSDM affiliates--<a href="https://collaborate.hms.harvard.edu/x/z4Gm">instructions
+      on how to login to your HMS Qualtrics account:
```
Comment (sapnamysore) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: The anchor tag could probably start  from the collaborate link?(https://...)

File: qualtrics_link/templates/qualtrics_link/_unauthorized_message.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      https://collaborate.hms.harvard.edu/x/z4Gm</a>
+    </li>
+    <li>
+      HBS affiliates--<a href="https://hbs.qualtrics.com/ControlPanel/">login
+      to your HBS Qualtrics account: https://hbs.qualtrics.com/ControlPanel/</a>
```
Comment (sapnamysore) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: Same as above. The anchor tag should surround just the control panel link?

File: icommons_ext_tools/settings/base.py [hunk 3]
```diff
@@ -235,6 +235,11 @@
             'handlers': ['default'],
             'propagate': False,
         },
+        'icommons_common': {
+            'level': _DEFAULT_LOG_LEVEL,
+            'handlers': ['default'],
+            'propagate': False,
+        },
     }
 }
```
Comment (sapnamysore) [inline icommons_ext_tools/settings/base.py]: Clean this up?

File: icommons_ext_tools/requirements/local.txt [hunk 4]
```diff
@@ -4,6 +4,7 @@
 -r base.txt 
 
 # below are requirements specific to the local environment
-django-debug-toolbar==1.3.2
+django-debug-toolbar==1.6
```
Comment (sapnamysore) [inline icommons_ext_tools/requirements/local.txt]: As discussed offline, lets downgrade it to version 1.5 due to change in support for automatic setup

**Comment by sapnamysore** [comment]
- Minor clarification about the links and 
- a suggestion to downgrade the django-debug-toolbar library. As a workaround, I modified the debug setting on dev. The  following tech debt has more details and possible suggestions

File: qualtrics_link/templates/qualtrics_link/_unauthorized_message.html (hunk 1 above)
Comment (elliottyates) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: (As we discussed -- it was PO's preference)

Comment (elliottyates) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: (looks like the URL times out, as you mentioned -- good catch -- we'll push this through under the assumption that HMS will set up a page there eventually)

File: icommons_ext_tools/settings/base.py (hunk 3 above)
Comment (elliottyates) [inline icommons_ext_tools/settings/base.py]: Good catch

File: icommons_ext_tools/requirements/local.txt (hunk 4 above)
Comment (elliottyates) [inline icommons_ext_tools/requirements/local.txt]: Excellent catch ([TLT-2980](https://jira.huit.harvard.edu/browse/TLT-2980))

File: qualtrics_link/templates/qualtrics_link/_unauthorized_message.html (hunk 1 above)
Comment (cmurtaugh) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: I think that link will just time out when we're on VPN. There's some long-standing network-routing brokenness that means that all of the HMS network is unreachable when we're on VPN.

Comment (elliottyates) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: Ah, I didn't know that, but that might explain why some askup stuff wasn't working as expected at times. Thanks!

Comment (sapnamysore) [inline qualtrics_link/templates/qualtrics_link/_unauthorized_message.html]: Thanks  @cmurtaugh. Good to know

**Comment by sapnamysore** [comment]
Looks  good 👍

**Pull request merged**